client/html/src/Client/Html/Catalog/Lists/Standard.php

```diff
@@ -646,13 +646,14 @@ public function addData( \Aimeos\MW\View\Iface $view, array &$tags = [], &$expir
 		$page = min( max( $view->param( 'l_page', 1 ), 1 ), $pages );
 
 		$products = \Aimeos\Controller\Frontend::create( $context, 'product' )
+			->sort( $sort )
```

Comment:
Can you please add a comment why this was done?

Reply:
I updated the commit.